docker-compose.yml

```diff
@@ -4,24 +4,14 @@ services:
 
   backend:
     build:
-      context: .
-      dockerfile: ./backend/Dockerfile
+      context: ./backend
     container_name: django_backend
-    volumes:
-      - ./backend/:/app
     ports:
       - "8000:8000"
-    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
I believe `command` here overrides the `CMD` that is in the `Dockerfile`. So not sure if the `CMD` is even running when the container is spun up:

`gunicorn backend.wsgi --bind 0.0.0.0`

as that is being overwritten by 

`python manage.py runserver 0.0.0.0:8000`

So, updated the `CMD` to just be the `python` command.